develop/upload-test-results.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+echo "uploading test results from $(pwd)"
+
+for FILE in *.junit.xml; do
+    echo "uploading $FILE"
+
+    curl -i -X POST \
+      -H "Authorization: Token token=$BUILDKITE_ANALYTICS_TOKEN" \
```

Comment:
I'm a little surprised shellcheck doesn't complain that the variable reference should use {}. it's fine though